inter-rater_variability/02a_rootlets_to_spinal_levels.py

```diff
@@ -140,8 +151,31 @@ def project_rootlets_to_segmentation(im_rootlets, im_seg, im_intersect, rootlets
 
     # Loop across the rootlets levels
     for level in rootlets_levels:
-        # Get the list of slices where the level is present
-        slices_list = np.unique(np.where(im_intersect.data == level)[2])
+
+        if level_type == 'rootlets':
+            slices_list = np.unique(np.where(im_intersect.data == level)[2])
+
+        elif level_type == 'vertebral':
+            # Get the list of slices where the level is present
+            unique_slices = np.unique(np.where(im_intersect.data == level)[2])
+
+            # Extract relevant slices from unique_slices
```

Comment:
making it more specific

```suggestion
            # Keep only slices with a single value (e.g., `2`). This is done because slices between the adjacent vertebral levels might have two values; for example, a slice between C2-C3 vertebral levels might have values `2` and `3`.
            # By doing this, we are introducing a gap between vertebral levels --> discuss whether it is good or not
```